mscore/mixer.ui

```diff
@@ -5,12 +5,6 @@
   <property name="enabled">
    <bool>true</bool>
   </property>
-  <property name="floating">
-   <bool>false</bool>
-  </property>
-  <property name="features">
-   <set>QDockWidget::AllDockWidgetFeatures</set>
```

Comment:
Qt 4.8 lists this as obsolete already in the documentation, Qt 6 will finally remove it